### ISSUE TITLE
Alt-right copies to middle, not to right

### ISSUE THREAD

**Issue by AndreasTschritter** [WinMerge/winmerge#2222]
When doing a 3 way merge, pressing alt-right in the left pane copies the change below the cursor to the middle, not to the right, even though it is listed in the context menu on the "copy to right" item (which works as intended).

**Comment by sdottaka** [comment]
Do you believe that when you press Alt+Right in the left pane, you expect the content to be copied to the right pane? Unfortunately, the Alt+Right key is designed to copy to the adjacent pane on the right(middle pane in this case).

Or is it simply a note that the menu item "Copy to Right Alt+Right" in the context menu is incorrect?

**Comment by AndreasTschritter** [comment]
Yes, I believed that Alt+Right copies the content to the right pane, since the context menu says so.

It would be nice to have a key combo for Copy to Right in addition to Alt-Right for copying to the adjacent pane.

**Comment by meteorquake** [comment]
I think if I had a choice I would use Alt-Shift-Arrow(Left/Right) to copy to the far left/right. When done from the middle it would be the same as Alt-Arrow.
d
